libs/platform/settings.hpp

```diff
@@ -40,6 +40,12 @@ template <class Value>
   return StringStorage::Instance().GetValue(key, strVal) && FromString(strVal, outValue);
 }
 
+inline bool IsTrue(std::string_view key)
```

Comment:
Maybe this is more readable?
```suggestion
inline bool IsEnabled(std::string_view key)
```